resources/serverless/charts/webhook/values.yaml

```diff
@@ -95,6 +95,8 @@ initContainer:
     readOnlyRootFilesystem: true # Mandatory
 
 values:
+  featureFlags:
+    java17AlphaEnabled: false
```

Comment:
Do you think it makes sense to move `featureFlags` to it's own top level section? I know it's technically webhook configuration, but I feel it's important enough to have it's own top level section.

Reply:
I have a feeling that our values are distribution in the whole `values.yaml`. 
I think that it should be refactored in the future.